v1.22/digitalocean/PRODUCT.yaml

```diff
@@ -0,0 +1,8 @@
+vendor: DigitalOcean
+name: DigitalOcean Kubernetes
+version: v1.21.7-do.0
```

Comment:
This should be `1.22.7-do.0`.

Reply:
(i don't know for sure, but the wrong version listed here could lead CI checks on this PR to fail)